Makefile

```diff
@@ -192,3 +192,21 @@ install-geth:
  			go install -v github.com/ethereum/go-ethereum/cmd/geth@$(shell cat .gethrc); \
  			echo "Installed geth!"; true)
 .PHONY: install-geth
+
+dev-sepolia-up: pre-devnet
+	NETWORK="sepolia" PYTHONPATH=./bedrock-devnet $(PYTHON) ./bedrock-devnet/devnet/sepolia.py --monorepo-dir=. --l1-rpc=${L1_RPC_URL}
+.PHONY: dev-sepolia-up
+
+dev-sepolia-clean:
+	cd ./ops-bedrock && NETWORK="sepolia" docker compose --profile shutter down
+	docker image ls 'ops-bedrock*' --format='{{.Repository}}' | xargs -r docker rmi
+	docker volume ls --filter name=ops-bedrock --format='{{.Name}}' | xargs -r docker volume rm
+	rm -rf ./packages/contracts-bedrock/deployments/dev-sepolia
+	rm -rf ./.devnet
+	rm -f ./packages/contracts-bedrock/deploy-config/dev-sepolia.json
+	direnv reload
```

Comment:
you are right. I did this for convenience reasons because whenever we deploy we need to generate a new SALT. You can see how I generate a new salt in my top comment.